tests/vmi_configuration_test.go

```diff
@@ -1292,8 +1292,13 @@ var _ = Describe("Configurations", func() {
 				By("Expecting the VirtualMachineInstance console")
 				Expect(tests.LoginToFedora(agentVMI)).To(Succeed())
 
+<<<<<<< HEAD
 				By("Terminating guest agent and waiting for it to dissappear.")
 				Expect(console.SafeExpectBatch(agentVMI, []expect.Batcher{
+=======
```

Comment:
Oh a merge leftover.

Reply:
Oh no. Sorry I didn't notice this. Fixed it now.